cluster/gce/configure-vm.sh

```diff
@@ -386,7 +378,7 @@ function salt-node-role() {
 grains:
   roles:
     - kubernetes-pool
-  cbr-cidr: '$(echo "$MINION_IP_RANGE" | sed -e "s/'/''/g")'
+  cbr-cidr: 10.123.45.0/30
```

Comment:
But this might break any file-based pod which created by kubelet before we reconfig cbr0?